src/NServiceBus.RabbitMQ/Receiving/MessageConverter.cs

```diff
@@ -83,10 +83,9 @@ string DefaultMessageIdStrategy(BasicDeliverEventArgs message)
             return properties.MessageId;
         }
 
-        static Dictionary<string, string> DeserializeHeaders(BasicDeliverEventArgs message)
+        static Dictionary<string, string> DeserializeHeaders(IDictionary<string, object> headers)
```

Comment:
`ToDictionary` is a LINQ extension method. I suppose a more precise commit message would have been "Remove ToDictionary usage from DeserializeHeaders"